sklearn/manifold/t_sne.py

```diff
@@ -672,6 +676,12 @@ def __init__(self, n_components=2, perplexity=30.0,
         self.method = method
         self.angle = angle
 
+        # TODO sklearn 0.20: remove this warning
+        if learning_rate == 200.0:
```

Comment:
Please read again https://github.com/scikit-learn/scikit-learn/pull/8397#issuecomment-281136788 and try to follow what I suggested.

If you don't like setting the default to None, use something else like `'default-changed-from-1000-to-200-in-0.19'` or something better.